### ISSUE TITLE
List Validation Rules - Wholistic Endpoint Review

### ISSUE THREAD

**Issue by janiksam** [voucherifyio/voucherify-openapi#128]
## What

ReportedBy: @janiksam 

### List Validation Rules

**GET** `/validation-rules`

#### Wholistic Endpoint Review

- [ ] Overall description
- [ ] Query parameters
- [ ] Headers
- [ ] Response description
- [ ] Response body
- [ ] Response example(s)
- [ ] All parameters'
    - [ ] examples
    - [ ] defaults
    - [ ] allowed values
    - [ ] properties
    - [ ] formats
- [ ] Hyperlinks

**Comment by MaciekVoucheirfy** [comment]
List validation rules - in applicable_to object need to correct - excluded FROM not in. Appears in excluded and object field.
In list validation rules in Rules Object within Campaign a typo error - campaign.redemptions.count.monthy should be monthLy

These errors are pretty much repeated throughout Validation Rules API calls not only in List call.

There is also a pretty big issue inside 'Create Validation Rules'. Talked about it with Karolina today and it is impossible to create a rule. It has some errors - all object names are string and order.amount for example should not be a string value.
Moreover there is some error in conditions object before you click on it, it says - Stringintegerboolean - no spaces, no possibility to choose anything. Not sure what causes the problem, probably will have to take a look at the code?

Generally this creation process looks a little bit complicated, not sure if we should place all the required in one place so the person that creates a rule is not confused. Some explanation with examples might be put below that. But I think that is something we will have to discuss in the future.

**Comment by janiksam** [comment]
You are right. The create validation rules endpoint description is still being developed and is under 🚧 . There are may ways to create a validation rule and to describe this requires deep understanding and testing of the endpoint.
